lib/resources/r4/related_person_patch.yaml

```diff
@@ -80,7 +80,16 @@ operations:
                 "system": "https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/351",
                 "code": "1152"
               }
-            ]
+            ],
+             "extension": [
```

Comment:
I think this needs some alignment, one space to the left.

Reply:
Done https://github.com/cerner/fhir.cerner.com/pull/650/commits/33e16ebe991c4ee4366849b4d4be826c338fd1f7#diff-a0c01bda612e4656e850590d65d36528f1caec8a70317c2ae4cb71d8095c9e7dR84-R92